.github/workflows/ci.yml

```diff
@@ -39,6 +39,11 @@ jobs:
         path: 'source'
         submodules: 'true'
 
+    - name: Setup NSIS Windows
+      if: matrix.os == 'windows-latest'
+      working-directory: ${{github.workspace}}
+      run: cp source\.github\nsis\FileAssociation.nsh "C:/Program Files (x86)/NSIS/Include"
```

Comment:
Use `${env:ProgramFiles(x86)}` instead of `C:/Program Files (x86)`

Reply:
does it matter ? The install path is standard on the CI machines.

Reply:
It's just cleaner and more robust in case github changes the install path (unlikely I know but it does not hurt to make the modification on our side)